node_modules/oae-principals/lib/init.js

```diff
@@ -56,8 +56,14 @@ module.exports = function(config, callback) {
 var _ensureSchema = function(callback) {
     // Both user and group information will be stored inside of the Principals CF
     Cassandra.createColumnFamilies({
-        'Principals': 'CREATE TABLE "Principals" ("principalId" text PRIMARY KEY, "tenantAlias" text, "displayName" text, "description" text, "email" text, "emailPreference" text, "visibility" text, "joinable" text, "lastModified" text, "locale" text, "publicAlias" text, "largePictureUri" text, "mediumPictureUri" text, "smallPictureUri" text, "admin:global" text, "admin:tenant" text, "notificationsUnread" text, "notificationsLastRead" text, "acceptedTC" text, "createdBy" text, "created" timestamp, "deleted" timestamp)',
-        'PrincipalsByEmail': 'CREATE TABLE "PrincipalsByEmail" ("email" text PRIMARY KEY, "principalId" text)'
+        'Principals': 'CREATE TABLE "Principals" ("principalId" text PRIMARY KEY, "tenantAlias" text, "displayName" text, "description" text, "email" text, "emailPreference" text, "visibility" text, "joinable" text, "lastModified" text, "locale" text, "publicAlias" text, "largePictureUri" text, "mediumPictureUri" text, "smallPictureUri" text, "admin:global" text, "admin:tenant" text, "notificationsUnread" text, "notificationsLastRead" text, "acceptedTC" text, "createdBy" text, "created" timestamp, "deleted" timestamp, "emailVerified" boolean, "emailChange" text)',
+
+        // Map an email address to user ids. An e-mail address can be used by *multiple* users
+        'PrincipalsByEmail': 'CREATE TABLE "PrincipalsByEmail" ("email" text, "principalId" text, PRIMARY KEY ("email", "principalId")) WITH COMPACT STORAGE',
```

Comment:
I thought that a `COMPACT STORAGE` table would actually require a 3rd column "value", such that:

`email` would be the row key (partition key)
`principalId` would be the column name (cluster key)
`value` would be the value that gets stored in the column

Aside from that, is there any reason we need to make this `COMPACT STORAGE` as opposed to the more modern Cassandra storage format that would be able to support more additions (e.g., new columns, secondary index, etc...) in the future if needed?